etl/steps/data/garden/emissions/2023-10-10/net_zero_tracker.meta.yml

```diff
@@ -0,0 +1,32 @@
+definitions:
+  common:
+    description_key:
+      - "The net zero status of a country can be one of the following:
+        - Achieved (externally validated)
+        - Achieved (self-declared)
+        - In law
+        - In policy document
+        - Declaration / pledge
+        - Proposed / in discussion"
+      - The inclusion criteria for net-zero commitments may vary from country to country. For example, the inclusion of international aviation emissions, or the acceptance of carbon offsets.
+    description_from_producer: "More detail about each indicator is available in the Tracker Codebook: https://docs.google.com/document/d/13bGE8FCOeF5QweoOvyMPZi3XbZrGLEPL2aM04LuZQcc/edit?usp=sharing "
+    unit: ""
+    short_unit: ""
+
+dataset:
+  update_period_days: 365
+
+tables:
+  net_zero_tracker:
+    variables:
+      net_zero_status:
+        title: Status of net-zero carbon emissions targets
+        description_short: ""
```

Comment:
No `description_short`, when it's required

Reply:
Good point, I think `description_short` should complement the title. In this case, I think the title is descriptive enough. But I see that in the guidelines, this field is required. I'll discuss this with Joe.